cubical/plots/gainsols.py

```diff
@@ -202,7 +202,7 @@ def _make_ap_plot(ax, ax2, freq, x1, x2, corrs, legend):
                                     ("c", ph1),  ("y", ph2))
                                ], loc="upper center", ncol=2, fontsize=options.font_size)
 
-    for iant, (ant, (time, freq, g00, g01, g10, g11)) in enumerate(sols.items()[:max_sols]):
+    for iant, (ant, (time, freq, g00, g01, g10, g11)) in enumerate(list(sols.items())[:max_sols]):
```

Comment:
I think:
```python
for iant, (ant, (time, freq, g00, g01, g10, g11)) in enumerate(list(sols.items())[:max_sols]):
```
might be worth replacing with:
```python
for iant, (ant, (time, freq, g00, g01, g10, g11)) in enumerate(sols.items()):

    if iant == max_sols:
        break
```
This is a minor nitpick.

Reply:
I have a stylistic aversion to ``break``, so I'll tie myself into loops sometimes (hehe) to avoid it... but I agree, in this case it simplifies the reading.